### PR TITLE
feat(sentry): update and connect sentry configuration for the project

### PR DESCRIPTION
**Pull request by tourlane-a-lanovyy**
## Context/Change

Add sentry configuration for tourlane sentry space

### Checklist

- [x] Ensure you have written good tests :heart:
- [x] Request reviewers (they would love to see your code :policeman:)
- [x] Use labels  🏷  (if applicable)
- [x] Test your code in appropriate environment i.e development/staging/production

## Reference

TRIP-7046

### CODE REVIEW

File: apps/web/next.config.js [hunk 1]
```diff
@@ -255,6 +255,8 @@ const nextConfig = {
 
 const sentryWebpackPluginOptions = {
   silent: true, // Suppresses all logs
+  dryRun: process.env.SENTRY_ENVIRONMENT !== "production",
```
Comment (haffla) [inline apps/web/next.config.js:258]: What is dryRun doing? What if environment is staging? We want to enable Sentry for staging and environment.

File: apps/web/sentry.client.config.js [hunk 2]
```diff
@@ -3,12 +3,14 @@
 // https://docs.sentry.io/platforms/javascript/guides/nextjs/
 import * as Sentry from "@sentry/nextjs";
 
-const SENTRY_DSN = process.env.NEXT_PUBLIC_SENTRY_DSN;
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
```
Comment (haffla) [inline apps/web/sentry.client.config.js:6]: Is it ever `SENTRY_DSN`? Isn't it always `NEXT_PUBLIC_SENTRY_DSN`?

**Review by haffla — COMMENTED** [review]
Looking good. Just some comments / questions.

File: apps/web/next.config.js (hunk 1 above)
Comment (tourlane-a-lanovyy) [reply apps/web/next.config.js:258]: It's used for release and source maps management integration for sentry and webpack. We don't need it for dev and stage(?), only for production.
Another way it requires `authToken` to be provided

> Attempts a dry run (useful for dev environments). Defaults to false, but may be automatically set to true in development environments by some framework integrations (Next.JS, possibly others).

Comment (haffla) [reply apps/web/next.config.js:258]: Yea I would say let's also enabled it for staging too

File: apps/web/sentry.client.config.js (hunk 2 above)
Comment (tourlane-a-lanovyy) [reply apps/web/sentry.client.config.js:6]: It looks like some leftovers from forking the main repo
https://github.com/calcom/cal.com/search?q=SENTRY_DSN+in%3Afile&type=Code

We can clear it out; however, we will have more project differences which could lead to a harder merge from the main repo

Comment (haffla) [reply apps/web/sentry.client.config.js:6]: OK whatever. You can remove it or leave it. I think it won't be hard to solve this merge conflict.

**Review by haffla — APPROVED** [review]

**Pull request merged**